samples/dotnetapp/utils/utils.csproj

```diff
@@ -0,0 +1,7 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
Is this intentionally different than the other two projects?

Reply:
The other two need to be netcoreapp. This one has the potenial to be netstandard. It aligns with our netstandard-first guidance for libraries.